database_admin/migrations/028_update_delete.up.sql

```diff
@@ -0,0 +1,40 @@
+CREATE OR REPLACE FUNCTION delete_system(inventory_id_in varchar)
```

Comment:
we should need to add more tests for whole this package-analysis related logic. I am afraid of other bugs that will be found later without better coverage

Reply:
We have average coverage, the problem is we have only example data. Real data from vmaas is different.